IPs/RSKIP454.md

```diff
@@ -0,0 +1,59 @@
+---
+rskip: 454
+title: Add support to bitcoin blocks with chain work up to 32 unsigned bytes
+created: 25-Oct-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          | 454                                                                   |
+| :------------ |:----------------------------------------------------------------------|
+|**Title**      | Add support to bitcoin blocks with chain work up to 32 unsigned bytes |
+|**Created**    | 25-Oct-24                                                             |
+|**Author**     | NC                                                                    |
+|**Purpose**    | Sca                                                                   |
+|**Layer**      | Core                                                                  |
+|**Complexity** | 1                                                                     |
+|**Status**     | Draft                                                                 |
+
+## Abstract
+
+This RSKIP proposes enabling RSK to accept Bitcoin blocks with chain work up to 32 unsigned bytes(256 bits). This change is necessary to ensure the bridge can process Bitcoin blocks with chain work values that exceed the current 12 unsigned bytes limit(96 bits).
```

Comment:
```suggestion
Currently, the bridge can process Bitcoin blocks with chain work values up to 12 unsigned bytes (96 bits).
This RSKIP proposes enabling RSK to increase this limit up to 32 unsigned bytes (256 bits).
```
something like that? wdyt?

Reply:
I prefer how it is, as my version emphasizes the inevitable necessity of this feature for the future by:
> ... This change is necessary to ensure the bridge can process Bitcoin blocks with chain work values that exceed the current 12 unsigned bytes limit(96 bits).